daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -99,7 +99,7 @@ def enable(name=None, verbose=True):
         for key in _mapping:
             do_patch(key)
     if verbose and sys.stderr is not None:
-        sys.stderr.write("Intel(R) DAAL solvers for sklearn enabled: "
+        sys.stderr.write("Intel(R) Data Analytics Acceleration Library (Intel DAAL) solvers for sklearn enabled: "
```

Comment:
Intel(R) Data Analytics Acceleration Library (Intel(R) DAAL)
looks like the second '(R)' sign is necessary according
http://prod-namesdb.intel.com/search.asp with search string 'DAAL':
The approved usage is: Intel® Data Analytics Acceleration Library (Intel® DAAL)

Reply:
Done!